### PR TITLE
Feat  device account 2

### PR DESCRIPTION
**Pull request by UncleSamtoshi**
- add feature flag
- clean up level naming
- use new device account mutation

### CODE REVIEW

File: app/graphql/feature-flags-context.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React, { useState, createContext, useContext, useEffect } from "react"
```
Comment (nicolasburtey) [inline app/graphql/feature-flags-context.tsx:1]: it's not really a graphql related feature?

Comment (nicolasburtey) [reply app/graphql/feature-flags-context.tsx:1]: looking at the filename

File: app/graphql/level-component.tsx [hunk 2]
```diff
@@ -25,17 +25,17 @@ gql`
 
 export const LevelContainer: React.FC<PropsWithChildren> = ({ children }) => {
   const isAuthed = useIsAuthed()
-  const isLevel0 = isAuthed
+  const isAtLeaseLevelZero = isAuthed
```
Comment (nicolasburtey) [inline app/graphql/level-component.tsx:28]:
```suggestion
  const isAtLeastLevelZero = isAuthed
```

**Comment by nicolasburtey** [comment]
some comments on https://github.com/GaloyMoney/galoy-mobile/pull/2119

File: app/graphql/feature-flags-context.tsx (hunk 1 above)
Comment (UncleSamtoshi) [reply app/graphql/feature-flags-context.tsx:1]: Any suggestions on where to put it? Maybe under the `config` folder or just the components.

**Pull request merged**